projects/tutorials/pointnav_ithor_rgb_ddppo.py

```diff
@@ -341,5 +341,5 @@ def test_task_sampler_args(
         res["loop_dataset"] = False
         res["env_args"] = {}
         res["env_args"].update(self.ENV_ARGS)
-        res["env_args"]["x_display"] = "10.0"
+        res["env_args"]["x_display"] = "0.0"
```

Comment:
We should probably grab the `x_display` value from shell `DISPLAY` variable and only use "0.0" as a fallback if it isn't set.

Reply:
I think we can also leave as None and let unity use the first display... I'll check it.